x-pack/test/functional/services/ml/navigation.ts

```diff
@@ -38,7 +38,7 @@ export function MachineLearningNavigationProvider({
 
     async navigateToAlertsAndAction() {
       await PageObjects.common.navigateToApp('triggersActions');
-      await testSubjects.click('alertsTab');
+      await testSubjects.click('rulesTab');
       await testSubjects.existOrFail('alertsList');
```

Comment:
@ymao1 maybe you also want to rename `alertsList` test subject 🙂

Reply:
This PR is for UI changes only so I tried to minimize variable name changes as much as possible. `alertsTab` only changed the `rulesTab` because I updated the client route from `alerts` to `rules`. We will tackle aligning variable names to terminology at a later date (that PR will be huge!)